compose-files/aquarius.yml

```diff
@@ -0,0 +1,17 @@
+version: '2.1'
+services:
+  aquarius:
+    image: oceanprotocol/aquarius:${OCEAN_VERSION:-stable}
+    ports:
+    - 5000:5000
+    networks:
+      backend:
+        ipv4_address: 172.15.0.15
+    depends_on:
+    - keeper-contracts
```

Comment:
Sure! I will review dependencies, but IMHO I would remove all the `depends_on` and try to guarantee these dependencies in the components because `depends_on` does not guarantee that the service is ready in any way (https://docs.docker.com/compose/compose-file/#depends_on). Some opinion about this?